### DISCUSSION TITLE
Lotería

### DISCUSSION THREAD

**Discussion by zebnat** [zebnat/morpho-dimension#22] [Ideas]
Comprar boletos con combinaciones de números de lotería que al finalizar la semana otorgará premios.

## Ejemplo

El usuario `soyLudopataAyuda` decide comprar 3 boletos.

`31502` `10414` `52015`

Al finalizar la semana el número de lotería ganador es `92414`

Uno de los boletos tiene premio por los 3 últimos números `414`

## Tabla de premios: (ejemplo requiere calcular con propiedad y dependerá del precio de cada boleto)

5 Cifras 25,000,000 macromonedas
4 Cifras 1,000,000 macromonedas
3 Cifras 300,000 macromonedas
2 Cifras 20,000 macromonedas
1 Cifra Devolución

Al final de la semana se citarán los ganadores por grupos de cifras.

**Comment by Darkusito** [comment]
Omg si, y qué el ganador sea revelado en algún lugar del apartado de comunidad, para estar al tanto. Excelente idea la vdd
